portfolio/src/main/webapp/script.js

```diff
@@ -66,13 +66,31 @@ class Portfolio {
         let parentList = document.getElementById("comments");
         comments.forEach(comment => parentList.appendChild(this.createListElement(comment)));
     }
+
+    /**
+    * Deletes all comments from the datastore.
+    */
+    async deleteComments() {
+        let path = '/delete-data';
+        let res = await fetch(path, { method: 'POST' });
+
+        /* Check for errors in the HTTP response and alert the user. */
+        if (res.status != 200) {
```

Comment:
What is status 200? Does it correspond to a particular error? If so, the alert could contain details for this error.

Reply:
Sorry, I noticed it's != 200, which means 200 must be the success. :)

Is it possible to check for other error codes and produce different errors? This might be out of scope.